composer.json

```
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^7.2.5",
+        "php": "^7.2 | ^7.3 | ^7.4 | ^8.0",
```

This can be simplified to the following to support 7.3 and 7.4. :+1:
```suggestion
        "php": "^7.2.5 | ^8.0",
```